change/@ni-nimble-components-8f211d7c-8a0f-4344-bb74-a82a49224602.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Could you add docs for the new validity field in the table's storybook? We have a parameter there that lists all the ways a table can be invalid.

Reply:
Done